web/portal/brand/src/entities/DdiProvider/Form.tsx

```diff
@@ -34,6 +34,7 @@ const Form = (props: EntityFormProps): JSX.Element => {
         'description',
         'proxyTrunk',
         // 'mediaRelaySets', @todo missing field
```

Comment:
Remove the comment please

Reply:
I'm leaving this like that because it was there before.

I have an incoming task for cleaning up things. There are still some fields in db to be removed, as well as some references in back and front.